src/client/nosto.d.ts

```diff
@@ -3769,7 +3769,7 @@ declare const api: {
      * require programmatic access to the Nosto request builder.
      * <br/><br/>
      * If your site is a single-page application atop a framework such as React, Vue, Angular or
-     * the likes, and  you are implementing Nosto, you must use the {@link defaultSession}
+     * the likes, and  you are implementing Nosto, you must use the {@link API.defaultSession}
      * method.
      *
      * @param {Object} flags a set of flags to customise to request behaviour (eg. {"includeTagging":true}
```

Comment:
consider adding correct type here instead of object?

Reply:
should I drop the type instead, since it is taken from TS anyway?

Reply:
Yes. It's better to fix the source rather

Reply:
Maybe let's fix it and come back to this PR?